Sources/SwiftDiagnostics/DiagnosticsFormatter.swift

```diff
@@ -195,4 +406,14 @@ struct ANSIAnnotation {
   static var normal: ANSIAnnotation {
     self.init(color: .normal, trait: .normal)
   }
+
+  /// Annotation used for the outline and line numbers of a buffer.
+  static var bufferOutline: ANSIAnnotation {
+    ANSIAnnotation(color: .cyan, trait: .normal)
+  }
+
+  /// Annotation used for highlighting source text.
+  static var sourceHighlight: ANSIAnnotation {
+    ANSIAnnotation(color: .white, trait: .underline)
```

Comment:
Will this also print the text as white in a terminal with white background and thus make it invisible? Shouldn’t it be

```suggestion
    ANSIAnnotation(color: .normal, trait: .underline)
```

Reply:
Yeah, I wanted to do that, but if I. use `.normal` here, Terminal doesn't show the underline.

Reply:
Hmm, but having invisible test in a white terminal seems like a really bad problem to me (I’m one of those weirdos who doesn’t use a dark terminal 😆).

Reply:
Addressed this in https://github.com/apple/swift-syntax/pull/1365